.travis.yml

```diff
@@ -15,7 +15,7 @@ install:
       DDG::SpiceBundle::OpenSourceDuckDuckGo
       DDG::FatheadBundle::OpenSourceDuckDuckGo
       DDG::LongtailBundle::OpenSourceDuckDuckGo
-  - dzil authordeps | grep -ve '^\W' | xargs -n 5 -P 10 cpanm --quiet --notest
+  - dzil authordeps | grep -ve '^\W' | xargs -n 5 -P 10 cpanm --mirror http://cpan.org --mirror http://duckpan.org --quiet --notest
```

Comment:
I mean why's it in this PR? The change seems a bit off-topic for the removal of the PM requirement.